force-app/main/default/classes/SOQL.cls

```diff
@@ -100,12 +95,11 @@ public with sharing class SOQL {
      * @return A map of Task Ids to Task records.
      * Update the corresponding test class and testGetTasksDueNextWeek method to cover this method
      */
-    public static Map<Id, Task> getTasksDueNextWeek() {
-    //  The body of the method
-    Date today = Date.today();
-    Date sevenDaysFromNow = today.addDays(7);
-    Map<Id, Task> dueTasks = new Map<Id, Task>([SELECT Id, ActivityDate FROM Task WHERE ActivityDate >= :today AND ActivityDate <= :sevenDaysFromNow]);
-    return dueTasks;
+    public static Map<Id, Task> getDueTaskRecords() {
+        Date today = Date.today();
+        Date sevenDaysFromNow = today.addDays(7);
+        Map<Id, Task> dueTasks = new Map<Id, Task>([SELECT Id, ActivityDate FROM Task WHERE ActivityDate >= :today AND ActivityDate <= :sevenDaysFromNow]);
```

Comment:
This works. There are also some relative date formula that we did not review, but would help here. `ActivityDate <= NEXT_N_DAYS:7`